meta_tests/conftest_test.py

```diff
@@ -1,32 +1,31 @@
 from unittest import TestCase
 
+import pytest
+
 from conftest import is_skippable
-from mock import Mock
+from mock import Mock, patch
+
+class DTestConfigMock():
+    def __init__(self):
+        self.execute_upgrade_tests = False
+        self.execute_upgrade_tests_only = False
+        self.force_execution_of_resource_intensive_tests = False
+        self.only_resource_intensive_tests = False
+        self.skip_resource_intensive_tests = False
+        self.use_vnodes = False
+        self.use_off_heap_memtables = False
 
+    def set(self, config):
+        if config != "":
```

Comment:
nit: only `""` or anything not-empty, 
maybe `if config:` ?

Reply:
Only `""` is used in the code. Strictly speaking the provided strings in `config` need to fully correspond to the attribute names. But it might be overkill to implement enumerators or constants, so no typo can get there.